colorama/ansitowin32.py

```diff
@@ -69,14 +86,17 @@ def __init__(self, wrapped, convert=None, strip=None, autoreset=False):
         # to support the ANSI codes.
         conversion_supported = on_windows and winapi_test()
 
+        # Does this version of Win 10 support ANSI?
+        windows_ansi_support = on_windows and check_windows_ansi_support()
+
         # should we strip ANSI sequences from our output?
         if strip is None:
-            strip = conversion_supported or (not is_stream_closed(wrapped) and not is_a_tty(wrapped))
+            strip = conversion_supported or not windows_ansi_support or (not is_stream_closed(wrapped) and not is_a_tty(wrapped))
         self.strip = strip
 
         # should we should convert ANSI sequences into win32 calls?
         if convert is None:
-            convert = conversion_supported and not is_stream_closed(wrapped) and is_a_tty(wrapped)
+            convert = not windows_ansi_support or (conversion_supported and not is_stream_closed(wrapped) and is_a_tty(wrapped))
```

Comment:
Are you sure this should be "or"-ed and not "and"-ed? This way if the old condition with conversion_supported is True, then convert will be True even if windows_ansi_support is True, which (I think) is not the desired outcome. I think that there is the same problem with strip (but I may be mistaken). This is a difficult part.